src/EnergyPlus/CMakeLists.txt

```diff
@@ -976,9 +976,22 @@ if(BUILD_TESTING)
   add_executable(TestAPI_Runtime_C ${PROJECT_SOURCE_DIR}/tst/EnergyPlus/api/TestRuntime.c)
   target_link_libraries(TestAPI_Runtime_C PRIVATE energyplusapi project_options project_fp_options)
   # target_compile_options(TestAPI_Runtime_C PRIVATE -Wall -Wextra -Wno-unused)
-
   add_test(NAME "API.TestRuntimeC" COMMAND TestAPI_Runtime_C -d "${TEST_DIR}" -w "${EPW_FILE}" -D "${IDF_FILE}")
 
+  set(TEST_DIR "${PROJECT_BINARY_DIR}/tst/api/runtimeresetpython")
```

Comment:
I added 4 tests:
* stateDelete in both C and python
* stateReset in both C and python

Without the fix the C ones pass, the python ones are exhibiting the segfault (tested ubuntu 20.04)

Reply:
I always love TDD.